core/input/input.cpp

```diff
@@ -149,6 +149,9 @@ void Input::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_action_just_pressed", "action"), &Input::is_action_just_pressed);
 	ClassDB::bind_method(D_METHOD("is_action_just_released", "action"), &Input::is_action_just_released);
 	ClassDB::bind_method(D_METHOD("get_action_strength", "action"), &Input::get_action_strength);
+	ClassDB::bind_method(D_METHOD("get_action_raw_strength", "action"), &Input::get_action_strength);
```

Comment:
@groud Do you remember if this was supposed to be bound (or we can re-decide now I guess)? It seems that I bound this incorrectly, but I'm not even sure that it should be bound. If so, this needs to be `get_action_raw_strength`, and also a string needs to be added to the if statement in `get_argument_options`.

Reply:
Ah yeah, it's indeed bound incorrectly.

I don't think accessing the raw value is needed anymore, now that we have get_vector() or get_axis(). But I guess it does not hurt that much to expose it I guess, if anyone want to implement their own deadzone system.

Reply:
Fixed in #50789